README.md

```diff
@@ -12,6 +12,6 @@
 Apache 2.0
 
 [travis_img]: https://img.shields.io/travis/mongodb-js/mongodb-js-cli.svg
```

Comment:
The travis_img is still busted.

Looking at https://travis-ci.org/mongodb-js/cli and clicking on the status image, I see that it should be

https://travis-ci.org/mongodb-js/cli.svg?branch=master